src/pkt_lldp.erl

```diff
@@ -126,7 +161,13 @@ map(port_id, mac_address)      -> ?PORT_ID_MAC;
 map(port_id, network_address)  -> ?PORT_ID_NW;
 map(port_id, interface_name)   -> ?PORT_ID_IFNAME;
 map(port_id, agent_circuit_id) -> ?PORT_ID_AGENT_CIRC_ID;
-map(port_id, locally_assigned) -> ?PORT_ID_LOCALLY.
+map(port_id, locally_assigned) -> ?PORT_ID_LOCALLY;
+
+map(org_specific, 'ieee 802.1') -> ?IEEE_8021;
+map(org_specific, 'ieee 802.3') -> ?IEEE_8023;
+map(org_specific, 'tr-41') -> ?TIA_TR41;
+map(org_specific, profibus) -> ?PROFIBUS;
+map(org_specific, gmbh) -> ?GMBH.
 
```

Comment:
insufficient for decoding(or, no decoding?).
you must be add encoding like follows:

```erlang
map(org_specific, ?IEEE_8021) -> 'ieee 802.1';
map(org_specific, ?IEEE_8023) -> 'ieee 802.3';
map(org_specific, ?TIA_TR41) -> 'tr-41';
map(org_specific, ?PROFIBUS) -> profibus;
map(org_specific, ?GMBH) -> gmbh.
```